aries-site/src/examples/templates/FilterControls/FiltersLayer.js

```diff
@@ -35,21 +38,22 @@ export const FiltersLayer = () => {
       as="section"
       position={size !== 'small' ? 'right' : undefined}
       full={size === 'small' ? true : 'vertical'}
-      modal
       onClickOutside={() => closeLayer()}
       onEsc={() => closeLayer()}
+      {...layerProps}
     >
       <Box
         fill="vertical"
         gap="medium"
         pad={{ vertical: 'medium' }}
         width={{ min: size !== 'small' ? 'medium' : undefined }}
+        {...containerProps}
       >
         <Header pad={{ horizontal: 'medium' }}>
           <Heading margin="none">Filters</Heading>
           <Button icon={<FormClose />} onClick={() => closeLayer()} />
         </Header>
-        <Box pad="medium" overflow="auto" flex>
+        <Box pad="medium" overflow="auto" flex {...contentProps}>
```

Comment:
Should content props be here? I also see it on the contained FormField which makes more sense to me

Reply:
Or vice versa, but it was confusing to me that it's on both

Reply:
I can see the confusion. The same naming is coming from two different instances and have two different targets.

In this instance, `contentProps` is coming from the `layerProps` object. The use in the FormField comes from the `filtersConfig` array objects, but also uses the name `contentProps` which is how the Grommet docs for FormField refer to it.

Open to suggestions.

Reply:
Got it, thanks for clarifying. Skimming the code I got a little lost. I like the idea of keeping with Grommet convention for FormField. For it be too verbose to use `formFieldContainerProps` or `inputsContainer` or something like this?

Reply:
Adjusted the name so that the main content Box uses `mainProps`. Does that work any better for you?

Reply:
After discussion, reverting back to `contentProps`.